terraform/environment-pipelines/buildspec-install-build-tools.yml

```diff
@@ -36,13 +36,10 @@ phases:
       - echo "Installing platform-helper from reference ${REQUIRED_VERSION}"
       ## Clone platform-tools:
       - echo -e "\nGet CodeStar connection details"
-      - codestar_connections=$(aws codestar-connections list-connections --provider-type GitHub --query "Connections[? ConnectionStatus == 'AVAILABLE']")
-      - aws_account=$(echo "$codestar_connections" | jq -r ".[0].OwnerAccountId")
-      - codestar_arn=$(echo "$codestar_connections" | jq -r ".[0].ConnectionArn")
-      - codestar_connection_id=$(echo "${codestar_arn##*/}")
-      - export GIT_CLONE_BASE_URL="https://codestar-connections.eu-west-2.amazonaws.com/git-http/$aws_account/eu-west-2/$codestar_connection_id/uktrade"
       - git config --global credential.helper '!aws codecommit credential-helper $@'
       - git config --global credential.UseHttpPath true
+      - codestar_connection_id=$(echo "${CODESTAR_CONNECTION_ARN##*/}")
```

Comment:
Be nice if this `codestar_connection_id` was provided as part of the `data.external.codestar_connections.result`, but that's somewhat outside the scope of this PR.